examples/integration/iOSApp/Test/SwiftUnitTests/BUILD

```diff
@@ -3,6 +3,7 @@ load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
 
 ios_unit_test(
     name = "iOSAppSwiftUnitTests",
+    args = ["--command_line_args=-AppleLanguages,(en)"],
```

Comment:
How does this interact with custom scheme args? I would assume the custom scheme ones override these?

Reply:
I will add a test case for this scenario